sdk/containerregistry/perf-tests/container-registry/package.json

```diff
@@ -27,9 +27,9 @@
     "build": "tsc -p .",
     "build:samples": "echo skipped",
     "build:test": "echo skipped",
-    "check-format": "prettier --list-different --config ../../../../.prettierrc.json --ignore-path ../../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "check-format": "prettier --list-different --config ../../../../.prettierrc.json --ignore-path ../../../../.prettierignore \"test/**/*.ts\" \"*.{js,json}\"",
```

Comment:
why is `\"src/**/*.ts\"` removed?

Reply:
Because perf-tests don't and won't have a `src/` folder and having this path was making the CI fail. I asked @HarshaNalluru about removing this and he mentioned that any extra code we want to put in the perf-tests will go inside `tests/utils/`